frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/activity-card/activity-card.tsx

```diff
@@ -51,12 +74,24 @@ const RecentEvent = withDashboardResources(
 );
 
 const OngoingActivity = withDashboardResources(
-  ({ watchPrometheus, stopWatchPrometheusQuery, prometheusResults }: DashboardItemProps) => {
+  ({
+    watchPrometheus,
+    stopWatchPrometheusQuery,
+    watchK8sResource,
+    stopWatchK8sResource,
+    resources,
+    prometheusResults,
+  }: DashboardItemProps) => {
```

Comment:
no need to specify `DashboardItemProps`
```suggestion
  }) => {
```